### PR TITLE
Added Vagrant setup and fixed the Dockerfile so it actually works

### PR DESCRIPTION
**Pull request by genebean**

### CODE REVIEW

**Comment by stahnma** [comment]
Why would we I want docker in vagrant vs just using docker? (I simply don't understand).

**Comment by genebean** [comment]
@stahnma I don't like the mess Docker can leave on my laptop so I generally run all my Docker stuff inside a VM. When I am finished or "think" I am finished with something Docker-related I will generally blow away the vm and bring it back up to see if I got it right.

**Comment by stahnma** [comment]
Ha, I feel the same way about Vagrant. :)

**Comment by genebean** [comment]
LOL. At least to clean up vagrant all I have to do is delete a folder or two... I always end up going back to Google to remember how to clean up after Docker.

**Comment by stahnma** [comment]
I always just had vbox crash my laptop. Either way, carry on.

**Comment by sbeaulie** [comment]
hey why not!?

File: Dockerfile [hunk 1]
```diff
@@ -8,24 +8,25 @@
...
-FROM jruby:1.7-jdk
+FROM jruby:9.1-jdk
+
+RUN mkdir -p /var/lib/vmpooler && mkdir /var/log/vmpooler
 
-RUN mkdir -p /var/lib/vmpooler
 WORKDIR /var/lib/vmpooler
 
-ADD Gemfile* /var/lib/vmpooler
+ADD Gemfile* /var/lib/vmpooler/
```
Comment (sbeaulie) [inline Dockerfile]: this * here failed for me when I had a Gemfile + a Gemfile.lock
I don't know why there is a * wildcard here?

**Pull request merged**